runhouse/servers/http/auth.py

```diff
@@ -75,6 +75,7 @@ def clear_cache(self, token: str = None):
 async def averify_cluster_access(
     cluster_uri: str,
     token: str,
+    write_only_access: bool = False,
```

Comment:
instead of a bool here, can just do `access_level_required` and pass `ResourceAccess.WRITE`

Reply:
yea good call